doc/source/conf.py

```diff
@@ -225,11 +225,24 @@
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
+
+switcher_version = version
+if ".dev" in version:
+    switcher_version = version.split(".0.dev")[0] + " (dev)"
```

Comment:
shouldn't this be a bit more generic?

Reply:
Can you clarify this question? 
(this only needs to handle our own `pd.__version__`, and produce something that matches what we define in the versions.json, so it doesn't need to be generic)